matminer/featurizers/tests/test_structure.py

```diff
@@ -864,7 +864,8 @@ def test_SOAP(self):
         self.assertAlmostEqual(df["SOAP_449"].iloc[1], 3.029167413711548, places=5)
 
     # todo: remove skip once https://github.com/materialsproject/pymatgen/pull/1548 is versioned into PMG
-    @unittest.skip()
+    @unittest.skip(reason="Pymatgen bv_sum interally broken, will be fixed "
+                          "in next PMG release")
```

Comment:
I would change this in two ways:

1. Write the specific PMG release in which this is expected to be fixed
2. Instead of skipping, uses a @unittest.expectedFailure decorator. Thus, when the problem is fixed, we will be forced to remove the decorator in order to get the tests to pass (otherwise, we might forget to remove the skip test).

Reply:
Perfect, thanks. Addressed in 32f6224ff8bf438cb9af20b1c0d53fb32163ebb7